Firestore/core/test/firebase/firestore/remote/serializer_test.cc

```diff
@@ -134,6 +136,76 @@ TEST_F(SerializerTest, EncodesBoolProtoToBytes) {
   }
 }
 
+TEST_F(SerializerTest, EncodesIntegersModelToProto) {
+  std::list<int64_t> testCases{0,
```

Comment:
I would imagine our default collection would just be a vector. List is inferior in every way unless:
  * You need pointer stability on mutation
  * Your lists will be huge and you expect to modify the middle a lot

Reply:
Done. (Not sure why I used a list here... especially given the use of vector in the method immediately above.)